agent/util-scripts/test-bin/rpm

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+args="${@}"
+echo "${0} ${args}" >> ${_testlog}
+
+# We are mocking out how the "rpm" command works by using certain names to
+# drive behaviors.  When not testing require-rpm itself, the special package
+# names are not used, and we assume success and exit with 0.  When the special
+# package names are used, we behave as described for each.
+
+_rpm="${3}"
```

Comment:
It seems like it would be worthwhile to sanity check that `${1}` is `--query` and that `${2}` is `--queryformat="%{EVR}"` (or similar).  It would serve to document what the first two arguments are, and might prevent confusion if this script gets invoked in something other than a query context.